src/style.css

```diff
@@ -0,0 +1,30 @@
+.main-content {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+/*  s*/
+.stars__item {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__item:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__item:nth-child(-n + 1),
+.stars--2 .stars__item:nth-child(-n + 2),
+.stars--3 .stars__item:nth-child(-n + 3),
+.stars--4 .stars__item:nth-child(-n + 4),
+.stars--5 .stars__item:nth-child(-n + 5) {
+  background: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
Those 2 values are not changing so you can get rid of them

Reply:
I had to shorten them into one line, otherwise tests were failed. All other recommendations are fixed.